website/siteConfig.js

```diff
@@ -13,7 +13,7 @@ function loadYaml(fsPath) {
   return parseYaml(fs.readFileSync(path.join(__dirname, fsPath), "utf8"));
 }
 
-const users = loadYaml("./data/users.yml");
+const users = loadYaml("./data/users.yml").sort(() => Math.random() - 0.5);
```

Comment:
This isn't fair: https://stackoverflow.com/a/18650169/2010616

Is it possible to use npm modules here? Could we just use lodash's shuffle or something?

Reply:
_but it's very cute_

Reply:
no need to require lodash, just for this.. I'll add a little helper function..

Reply:
From the same SO question..